uiowa/uiowa
# [Aggregator] Decode raw html in title

Raw HTML from the source is being displayed

<img width="946" alt="image" src="https://github.com/uiowa/uiowa/assets/49074566/600a2dc7-fce6-4ff3-b2c1-5352eccc40cd">






# How to test


```
ddev blt ds --site=provost.uiowa.edu
```
Check https://provost.uiowa.ddev.site/test-rss-feed and see that the HTML gets encoded
